clang/lib/Driver/ToolChains/Darwin.cpp

```diff
@@ -2923,22 +2923,47 @@ bool Darwin::isAlignedAllocationUnavailable() const {
   return TargetVersion < alignedAllocMinVersion(OS);
 }
 
-static bool sdkSupportsBuiltinModules(const Darwin::DarwinPlatformKind &TargetPlatform, const std::optional<DarwinSDKInfo> &SDKInfo) {
+static bool sdkSupportsBuiltinModules(
+    const Darwin::DarwinPlatformKind &TargetPlatform,
+    const Darwin::DarwinEnvironmentKind &TargetEnvironment,
+    const std::optional<DarwinSDKInfo> &SDKInfo) {
+  switch (TargetEnvironment) {
+  case Darwin::NativeEnvironment:
+  case Darwin::Simulator:
+  case Darwin::MacCatalyst:
+    // Standard xnu/Mach/Darwin based environments
+    // depend on the SDK version.
+    break;
+  default:
```

Comment:
It might cause a warning. Fixed in 0f1361baf650641a59aaa1710d7a0b7b02f2e56d.

@kazutakahirata I suggest mentioning (e.g. #102239) in your commit message, to notify fixups.

Reply:
Should we drop the `default` instead? I guess we could also drop the TargetEnvironment argument completely since it's not currently needed.

Reply:
Will do next time.  Thanks!

Reply:
Added that commit